### DISCUSSION TITLE
saving segmented/annotated point cloud

### DISCUSSION THREAD

**Discussion by nimafo** [isl-org/Open3D#5650] [Q&A]
Hi all,
I am using open3d for segmenting a point cloud, and I want to save the annotations(segment indices) in the written output file at the end. I see there are different output format options, but I wonder which is suitable for this purpose. 
<img width="426" alt="Screenshot 2022-11-03 at 18 51 22" src="https://user-images.githubusercontent.com/45439717/199995476-ba630b6b-4acc-4f5c-a991-68e8fc76e96b.png">

Thanks in advance,

**Comment by kgeorge-fission** [comment]
hi @nimafo, I'm trying to segment models using open3d. Could you tell me how you managed to segment your models?

**Reply by nimafo** [reply]
Hey @CriusFission,
Sure, I simply followed the steps in [this tutorial](https://towardsdatascience.com/how-to-automate-3d-point-cloud-segmentation-and-clustering-with-python-343c9039e4f5).
